examples/hello_world/CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ function(create_target executable)
     target_include_directories(vsomeip_hello_world_${executable} INTERFACE
         ${CMAKE_CURRENT_SOURCE_DIR}
     )
+
+    if(ENABLE_SIGNAL_HANDLING)
+        target_compile_definitions(vsomeip_hello_world_example INTERFACE VSOMEIP_ENABLE_SIGNAL_HANDLING)
+    endif()
```

Comment:
why is this needed, since the VSOMEIP_ENABLE_SIGNAL_HANDLING has already been set on the main cmakelists as an add_definitions() command